server.js

```diff
@@ -0,0 +1,3 @@
+var mathjaxServer = require("mathjax-server");
```

Comment:
Maybe this file should not be located in the root directory of the extension. A subfolder would be nice.